server/routes/induction/exemption/exemption/confirmExemptionController.test.ts

```diff
@@ -28,15 +27,15 @@ describe('ConfirmExemptionController', () => {
     req = {
       user: { username: 'a-dps-user' },
       params: { prisonNumber, journeyId },
-      session: {},
+      journeyData: {},
     } as unknown as Request
     res = {
       render: jest.fn(),
       redirect: jest.fn(),
       locals: { prisonerSummary },
     } as unknown as Response
 
-    getPrisonerContext(req.session, prisonNumber).inductionExemptionDto = undefined
```

Comment:
The changes to the controllers and their tests are essentially the same - everywhere where we used the prisonerContext and session to hold/reference the inductionExceptionDto has been changed to use req.journeyData.